sdk/src/feature_set.rs

```diff
@@ -554,6 +554,11 @@ pub mod enable_program_redeployment_cooldown {
     solana_sdk::declare_id!("J4HFT8usBxpcF63y46t1upYobJgChmKyZPm5uTBRg25Z");
 }
 
+pub mod prevent_commission_update_in_second_half_of_epoch {
+    // This is a feature-proposal *feature id*.  The feature keypair address is `ComffCcTMthDAx5AbFA4LeRzGRUB4Z1wNhXz354zvAmT`.
```

Comment:
I'm leaning towards this as a single-core-contributor feature actually. Perhaps we can raise this for discussion on an upcoming validator community call and Discord and see if there is any dissenting opinion.

Reply:
It seems like such a slam dunk for the community to vote on, since I think they'll all want it, but maybe that's the point: it's so uncontroversial that a single contributor can own it. Anyway, please bring it up during the call today